index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-	<title>Biblical Hebrew Reader Generator</title>
+	<title>Biblical OT Reader Generator</title>
```

Comment:
Please rename to "Hebrew Bible Reader Generator" (also below), to not assume a Christian background.